run.py

```diff
@@ -61,11 +61,12 @@ def parse_args():
         action="store_false",
         default=True,
     )
+    now = datetime.now()
     parser.add_argument(
         "--xlsx-name",
         help="Name of the XLSX file.",
         dest="xlsx_name",
-        default="",
+        default="trackit_aws_cost_report_{}_{}{}_{}{}".format(now.year, "0" if now.month < 10 else "", now.month, "0" if now.day < 10 else "", now.day),
```

Comment:
Please take advantage of the python date formatting feature:
`now.strftime('%Y-%m-%d')`

Reply:
Really good advice, thanks.